scripts/build.sh

```diff
@@ -51,13 +51,14 @@ if [[ -n "${TF_RELEASE}" ]]; then
 fi
 
 # Build!
+VERSION="1.64.0"
 echo "==> Building..."
 gox \
     -os="${XC_OS}" \
     -arch="${XC_ARCH}" \
     -osarch="${XC_EXCLUDE_OSARCH}" \
     -ldflags "${LD_FLAGS}" \
-    -output "pkg/{{.OS}}_{{.Arch}}/terraform-provider-ibm" \
+    -output "pkg/{{.OS}}_{{.Arch}}/terraform-provider-ibm_${VERSION}_{{.OS}}_{{.Arch}}" \
```

Comment:
These changes are required?

Reply:
No. Let me remove.

Reply:
@ujjwal-ibm I have removed the all changes in this file.